public/components/schemas/FormAnswerUpdate.yml

```diff
@@ -0,0 +1,9 @@
+type: object
+properties:
+  status:
+    type: string
+    enum:
```

Comment:
You should add a description for that field. The best would be to explain what each status mean but I don't really know what was the idea behind it.